web/src/components/table/use-cases/scores.tsx

```diff
@@ -33,14 +40,36 @@ export type ScoreFilterInput = Omit<
   "projectId" | "userId"
 >;
 
+function createFilterState(
+  userFilterState: FilterState,
+  omittedFilters: Record<string, string>[],
+) {
+  return omittedFilters.reduce((filterState, { key, value }) => {
+    return filterState.concat([
+      {
+        column: `${key}`,
+        type: "string",
+        operator: "=",
+        value: value,
+      },
+    ]);
+  }, userFilterState);
+}
+
 export default function ScoresTable({
   projectId,
   userId,
+  traceId,
+  observationId,
   omittedFilter = [],
+  hiddenColumns = [],
 }: {
   projectId: string;
   userId?: string;
+  traceId?: string;
+  observationId?: string;
   omittedFilter?: string[];
+  hiddenColumns?: string[];
 }) {
   const [paginationState, setPaginationState] = useQueryParams({
```

Comment:
we probably need observationId = Null here in order to only show trace scores on the trace level. otherwise we'd need to somehow identify them within the scores table on the trace level

Reply:
Decided to scope out of this PR as we'd require a `is null` filter on tables for this request, which we've scoped out into a separate feature request ticket on linear. For now: show all nested scores in the trace > scores tab.